lib/color.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+let chalk = require('chalk');
+
+module.exports = Object.assign(chalk, {
+  app: chalk.cyan,
```

Comment:
this needs to be `app: s => chalk.cyan(s), ` in order for it to be properly disabled